### PR TITLE
Automatically update SQS queue attributes

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

File: packages/sqs/lib/utils/sqsInitter.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import type { SQSClient } from '@aws-sdk/client-sqs'
+import { SetQueueAttributesCommand } from '@aws-sdk/client-sqs'
+import type { QueueAttributeName } from '@aws-sdk/client-sqs/dist-types/models/models_0'
```
Comment (jmontoyaa) [inline packages/sqs/lib/utils/sqsInitter.ts:3]: Is this import is correct? Should this include `/dist-types/models/models_0`?

Comment (kibertoad) [reply packages/sqs/lib/utils/sqsInitter.ts]: great catch! autoimport went wrong.
there were more of cases like this, fixed

**Review by jmontoyaa — APPROVED** [review]

File: packages/core/.eslintrc.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "@typescript-eslint/no-empty-interface": "warn",
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-use-before-define": "off",
-    "@typescript-eslint/no-non-null-assertion": "warn",
+    "@typescript-eslint/no-non-null-assertion": "off",
```
[the remark this answers is not in the document]
Comment (kibertoad) [reply packages/core/.eslintrc.json]: fixed

File: packages/sqs/.eslintrc.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
     "@typescript-eslint/no-empty-interface": "warn",
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-use-before-define": "off",
-    "@typescript-eslint/no-non-null-assertion": "warn",
+    "@typescript-eslint/no-non-null-assertion": "off",
```
Comment (CarlosGamero) [inline packages/sqs/.eslintrc.json:26]: Looks like we have the same here, maybe I am missing something and this is needed so please let me know if it is the case 🙇

Comment (kibertoad) [reply packages/sqs/.eslintrc.json:26]: will address in a follow-up PR

**Review by CarlosGamero — APPROVED** [review]

**Pull request merged**